### ISSUE TITLE
[Bug]: Focus ring shows on NavItem active

### ISSUE THREAD

**Issue by Robin1224** [fdnd-agency/atlas4045#135]
### Description

When clicking on a page in the navigation menu it shows the focus ring state for a split second until the page has transitioned

### Expected behavior

There should be no focus ring when clicking on a nav item

### Actual behavior

A focus ring shows for a split second when a nav item is clicked

### What device are you experiencing the problem on?

Laptop

### What operating system are you experiencing the problem on?

macOS

### What browsers are you experiencing the problem on?

Chrome
